backend/relayer-archive.md

```diff
@@ -0,0 +1,59 @@
+# Configure relayer "local mode"
```

Comment:
I'm sorry for being "that person", but this needs to be revamped based on #40 that has changes to configuration and commands. In particular, there are no special modes there, so this should be just a section in the readme.